### PR TITLE
README.md & CORE GET /api/topics complete

### PR DESCRIPTION
**Pull request by ginnytruong**

### CODE REVIEW

File: __tests__/app.tests.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+describe("GET /api/topics", () => {
+    test("returns status code 200 & array of objects with slug and description properties", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+            const topics = response.body.topics;
+            topics.forEach((topic) => {
+                expect(topic).toHaveProperty("slug");
```
Comment (ghost) [inline __tests__/app.tests.js:18]: You could also assert the data types here on the keys.

File: db/controller.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const { selectAllTopics } = require("./model");
+const fs = require("fs/promises");
```
Comment (ghost) [inline db/controller.js:2]: Why is this imported? Where are you using it?

File: db/model.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const db = require("./connection");
+
+exports.selectAllTopics = () => {
+    let topicsSqlStr = `SELECT * FROM topics`;
```
Comment (ghost) [inline db/model.js:4]: Unless the string will be added to, this should be a const.

**Review by ghost — COMMENTED** [review]
Nice start, a couple of things to consider. Good job, make sure you keep the rest of it nice and tidy :)

**Comment by ginnytruong** [comment]
Updated based on above feedback :)

**Pull request merged**